app/models/container_template.rb

```diff
@@ -1,5 +1,6 @@
 class ContainerTemplate < ApplicationRecord
   include CustomAttributeMixin
+  include ManageIQ::Providers::Kubernetes::ContainerManager::EntitiesMapping
```

Comment:
Do we want core code dependent on providers?
The other way around would be to have a template either moved to the provider or inherited there.

We don't need to get the dependency right the first time, since this is all very new, but we need to discuss what we want in the general case so I'm starting the discussion.

CC @ohadlevi